4.6.2/Dockerfile

```diff
@@ -9,6 +9,25 @@ RUN Add-WindowsFeature Web-Server; \
 
 ADD ServiceMonitor.exe /
 
+RUN powershell -Command \
+	#download and unzip Roslyn nupkg
+	wget  https://api.nuget.org/packages/microsoft.net.compilers.2.3.1.nupkg -OutFile c:\microsoft.net.compilers.2.3.1.zip ; \	
+	Expand-Archive -Path c:\microsoft.net.compilers.2.3.1.zip -DestinationPath c:\RoslynCompilers ; \
+
+	#ngen Roslyn binaries
+	start-Process C:\Windows\Microsoft.NET\Framework64\v4.0.30319\ngen.exe -ArgumentList 'install c:\RoslynCompilers\tools\csc.exe' -Wait ; \
```

Comment:
What happens here if ngen.exe returns a non-zero exit code?  Don't you need to check the exitcode on the start-process return value?  Would using the invocation operator '&' help out here?

Reply:
will change to use &